forc-plugins/forc-doc/src/render.rs

```diff
@@ -25,6 +25,7 @@ pub(crate) trait Renderable {
     fn render(self, render_plan: RenderPlan) -> Result<Box<dyn RenderBox>>;
 }
 /// A [Document] rendered to HTML.
+#[derive(Debug)]
```

Comment:
Should this be left in production?

Reply:
I think leaving it there will be okay since this is a rapidly growing program